readthedocs/oauth/migrations/0013_data_migration_for_remote_relation_model.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 2.2.16 on 2020-10-05 06:15
+
+import json
+from itertools import islice
+
+from django.db import migrations
+
+
+def move_data_to_remote_relations(apps, schema_editor):
+    RemoteRelation = apps.get_model('oauth', 'RemoteRelation')
+
+    def remote_relations_generator(relations, batch_size):
+        for relation in relations.iterator(chunk_size=batch_size):
+            relation.account_id = relation.remoterepository.account_id
+            relation.active = relation.remoterepository.active
+            relation.admin = relation.remoterepository.admin
+            relation.pub_date = relation.remoterepository.pub_date
```

Comment:
I think we should update the `modified_date` as well, to keep the original. Don't you think?

Reply:
Won't the ``modified_date`` change after we save the remote relation objects after the data migration?